crates/rpc/rpc-types/src/eth/pubsub.rs

```diff
@@ -104,8 +104,8 @@ pub enum Params {
     None,
     /// Log parameters.
     Logs(Box<Filter>),
-    /// New pending transaction parameters.
-    NewPendingTransactions(bool),
+    /// Boolean parameter for new pending transactions.
```

Comment:
nit: is this for re-use on other stuff? if so is it accurate to say "for new pending transactions"?

Reply:
technically this a standalone type, which is used as additional param in the subscription endpoint, but yes it's only used in new pending transactions subscription kind